### PR TITLE
Fixes #13850 - print help without --force for inactive scenario

### PR DESCRIPTION
**Pull request by mbacovsky**
`git diff HEAD^ -w` reveal better view on what has changed

### CODE REVIEW

**Comment by mbacovsky** [comment]
[test]

**Comment by mbacovsky** [comment]
Rebuilt job is green http://ci.theforeman.org/job/test_kafo_master_pull_request/184/

File: lib/kafo/kafo_configure.rb [hunk 1]
```diff
@@ -164,6 +164,10 @@ def self.exit_code
       self.exit_handler.exit_code
     end
 
+    def self.in_help_mode?
+      ARGV.include?('--help') || ARGV.include?('-h')
```
Comment (ares) [inline lib/kafo/kafo_configure.rb]: there's also a `--full-help`

**Comment by ares** [comment]
except one nit, it's ready

**Comment by ares** [comment]
@mbacovsky do you plan to work on this in near future? There's still the test issue present.

**Comment by mbacovsky** [comment]
@ares, yes, I hope I'll return back to this soon, thanks for your patience ;)

**Comment by mbacovsky** [comment]
@ares, shame on me for the tests. It was fixed, `--full-help' was added.

**Comment by ares** [comment]
Thanks @mbacovsky, merging!

**Pull request merged**